src/pages/lifetime/celebration/singlesday.tsx

```diff
@@ -0,0 +1,82 @@
+import Layout from '@/components/layout/Layout';
+import cookies from '@/lib/cookies';
+import TestimonialsSection from '@/components/home/TestimonialsSection';
+import Navbar from '@/components/layout/navbars/Navbar';
+import { MinimalFooter } from '@/components/layout/footers/MinimalFooter';
+import FAQSection from '@/components/shared/sections/FaqSection';
+import CtaSection from '@/components/shared/CtaSection';
+import PaymentSection from '@/components/lifetime/celebration/singles-day/PaymentSection';
+import FeatureSection from '@/components/lifetime/celebration/singles-day/FeatureSection';
+import HeroSection from '@/components/lifetime/celebration/singles-day/HeroSection';
+
+const SinglesdayCelebrationTemplate = ({
+  lang,
+  langJson,
+  navbarLang,
+  footerLang,
+  metatagsDescriptions,
+  textContent
+}) => {
```

Comment:
Add an interface for this props. Something like:
````
interface SinglesDayCelebrationTemplateProps {
lang: string;
langJson: HomePageText;
navbarLang: NavigationBarText;
footerLang: FooterText;
metatagsDescriptions: MetatagsDescriptions[];
textContent: SinglesDayCelebrationTemplateText;
}

const SinglesdayCelebrationTemplate = ({
  lang,
  langJson,
  navbarLang,
  footerLang,
  metatagsDescriptions,
  textContent
}: SinglesDayCelebrationTemplateProps): JSX.Element => {
````

Create a file inside assets/types called singlesDay.ts with the JSON types. You can use this extension to do it automatically: [Paste JSON as Code](https://marketplace.visualstudio.com/items?itemName=quicktype.quicktype)